modules/local/combine_tsv.nf

```diff
@@ -7,7 +7,7 @@ process COMBINE_TSV {
         'biocontainers/bioawk:1.0--hed695b0_5' }"
 
     input:
-    path(bin_summaries)
+    path(bin_summaries, stageAs: "bin_summaries/*.tsv")
```

Comment:
Yes, it updates the inputs to "1.tsv", "2.tsv" and so on. I did this to avoid filename clashing, because CheckM2 generates the summary as `${output_dir}/quality_report.tsv`.